upgrading/change-log.md

```diff
@@ -7,6 +7,39 @@ heading: Version 3.15.x
 
 <p class="release-date">2020</p>
 
+<h4>Fixes</h4>
+
+- In Marimekko chart, using `yAxisValueDecimal` now works properly when `setAdaptiveYMin` and `forceYAxisValueDecimals` are enabled.
+
+- Decimal and thousand separator attributes now work properly on Marimekko chart. 
+
+- For all 2D and 3D bar chart variants, the fontSize attribute now works properly when set with a numeric value. Earlier a JS error was thrown.
+
+- The task labels are now displayed properly in Gannt charts and are available on further horizontal scrolling. Previously, the task label overlapped with the vertical scroll bar. 
+
+- The attribute `seriesname` of a dataset is now working properly for charts rendered in IE 11 browser. Earlier, the usage of 1 {br} tag was considered as 2 {br} tags in IE 11 browser.
+
+- The event handlers `chartRollOver` and `chartRollOut`are now working properly when multiple charts are present on a web page. Earlier, upon multiple instances of hover over each other one of the charts stopped invoking the hover events and threw a JS error. 
```

Comment:
This is slightly incorrect. The situation was when multiple charts are present on page web with `chartRollOver` and `chartRollOut` events handled by a common handler function then upon multiple instances of hover charts threw JS error and stopped invoking the events.

Reply:
changed